### PR TITLE
bump version to 4.6.1, update changelogs, update copyright dates

### PR DESCRIPTION
**Pull request by sezero**
also fix a minor 'unused label' warning in lite builds.

### CODE REVIEW

File: src/effects.c [hunk 1]
```diff
@@ -445,7 +445,9 @@ void libxmp_process_fx(struct context_data *ctx, struct channel_data *xc, int ch
 			fxp <<= 4;
 			goto fx_setpan;
 		case EX_RETRIG:		/* Retrig note */
+		#ifndef LIBXMP_CORE_PLAYER
 		    fx_retrig:
+		#endif
```
Comment (AliceLR) [inline src/effects.c:450]: Please don't indent these macros; it makes them look like they're part of regular code flow and libxmp fairly consistently keeps them at indentation level 0 in other places.

**Comment by AliceLR** [comment]
Still have a few patches to finish up (.LIQ PR hopefully tonight and I'll try to finish up the rest over the weekend) but this looks OK.

File: src/effects.c (hunk 1 above)
Comment (sezero) [reply src/effects.c]: de-indented :)

**Comment by sezero** [comment]
> Still have a few patches to finish up (.LIQ PR hopefully tonight and I'll try to finish up the rest over the weekend) but this looks OK.
> 
> (s3m2liq converter and .LIQ tooling so I could fix the loader took up most of my week...)

Still be waiting. This is just to make things up-to-date

**Comment by sezero** [comment]
P.S.: This doesn't mark the release: see that the release dates in ChangeLogs aren't set yet.

**Review by AliceLR — APPROVED** [review]

**Pull request merged**